src/main/java/sigma/internship/petProject/entity/Game.java

```diff
@@ -0,0 +1,46 @@
+package sigma.internship.petProject.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity(name = "game")
+@Table(name = "game")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+public class Game {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "title", nullable = false, unique = true, length = 30)
+    private String title;
+
+    @Column(name = "description", nullable = false)
+    private String description;
+
+    @Column(name = "is_one_user_action", nullable = false)
+    private boolean isOneUserAction;
+
+    @Column(name = "winning", nullable = false)
+    private double winning;
+
+    @Column(name = "cost", nullable = false)
+    private double cost;
+
+    @Column(name = "create_date", nullable = false)
+    private LocalDateTime createDate;
+
+    @PrePersist
+    public void createEntity() {
+        createDate = LocalDateTime.now();
+    }
+}
```

Comment:
Please, add a blank line at the end of every code file.
Here you can find why it is necessary: 
https://stackoverflow.com/questions/2402593/whats-the-reason-for-leaving-an-extra-blank-line-at-the-end-of-a-code-file

Reply:
In PR it is shown only added/removed lines, while in IDEA there is everything with these blank lines, but in any case I will take it into consideration next time

Reply:
Yes, you are right, I double checked, everything is okay